selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java

```diff
@@ -324,85 +324,152 @@ public void waitTabIsNotPresent(String nameOfFile) {
     editor.waitTabIsNotPresent(nameOfFile);
   }
 
+  /** Waits visibility and performs click on the "Run Command" button. */
   public void clickOnRunButton() {
     seleniumWebDriverHelper.waitAndClick(runButton, REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
   }
 
+  /** Waits visibility and performs click on the "Cancel" button. */
   public void clickOnCancelCommandEditorButton() {
     seleniumWebDriverHelper.waitAndClick(cancelCommandEditorButton, REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
   }
 
+  /** Waits visibility and performs click on the "Save" button. */
   public void clickOnSaveButtonInTheEditCommand() {
     seleniumWebDriverHelper.waitAndClick(saveButtonInCommandEditor, REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
   }
 
   /**
-   * wait the command tab with specified name with unsaved status
+   * Waits until command tab with specified {@code commandName} is displayed with the unsaved
+   * status.
    *
-   * @param nameCommand name of tab command
+   * @param commandName title of the commands editor's tab
    */
-  public void waitTabCommandWithUnsavedStatus(String nameCommand) {
+  public void waitTabCommandWithUnsavedStatus(String commandName) {
     seleniumWebDriverHelper.waitVisibility(
-        By.xpath(format(ACTIVE_TAB_UNSAVED_FILE_NAME, nameCommand)), ELEMENT_TIMEOUT_SEC);
+        By.xpath(format(ACTIVE_TAB_UNSAVED_FILE_NAME, commandName)), ELEMENT_TIMEOUT_SEC);
   }
 
-  public void typeTextIntoNameCommandField(String nameCommand) {
-    seleniumWebDriverHelper.setValue(nameCommandField, nameCommand);
+  /**
+   * Types specified {@code commandName} to the "Command Name" field, and waits until typed text is
```

Comment:
I would mention updating of specified command immediately without additional notes which requires special attention and additional time: `Sets name of command to {@code commandName} and wait until it appears.`